tasks/import-listings/src/etl/transform/map.ts

```diff
@@ -52,7 +52,36 @@ export const defaultMap: RecordMap = {
   jurisdiction: (listing: Listing) => jsonOrNull(listing.jurisdiction),
   reserved_community_type: (listing: Listing) => jsonOrNull(listing.reservedCommunityType),
   units: (listing: Listing) => jsonOrNull(listing.units),
-  building_address: (listing: Listing) => jsonOrNull(listing.buildingAddress),
+  building_address: (listing: Listing) => {
+    const address = listing.buildingAddress
+
+    // if we don't have an address, ignore
+    if (!address) return
+
+    if (!address?.county) {
+      const jurisdiction = listing.jurisdiction
+
+      if (jurisdiction.name == "San Jose") {
+        address.county = "Santa Clara"
+      }
```

Comment:
Why is this needed here, if it's also part of the switch statement below? Seems repetitive, but maybe I'm missing something.

Reply:
You're right! I initially added the conditions as individual conditions just in case the test logic differed, but I ended up being able to put them all into a switch instead.  I'll remove it.